apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/extensions/InterfaceApi.java

```diff
@@ -38,18 +37,17 @@
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 
 /**
  * Provides access to the OpenStack Compute (Nova) Attach Interfaces API.
  */
 @Beta
-@Extension(of = ServiceType.COMPUTE, namespace = ExtensionNamespaces.ATTACH_INTERFACES)
+@Extension(of = ServiceType.COMPUTE, namespace = ExtensionNamespaces.INTERFACES)
 @RequestFilters(AuthenticateRequest.class)
 @Consumes(MediaType.APPLICATION_JSON)
 @Path("/servers")
-public interface AttachInterfaceApi {
+public interface InterfaceApi {
```

Comment:
But the problem might be that existing code will break. We might want to consider deprecating the original code instead, so it still works.